govcd/edgegateway.go

```diff
@@ -200,82 +200,82 @@ func (eGW *EdgeGateway) RemoveNATPortRule(networkHref, natType, externalIP, exte
 
 }
 
-// Temporary fix to support parsing network Id from HREF
-// TODO: remove this function when ruleId is available
-func extractObjectIDfromPath(locationPath string) (string, error) {
-	if locationPath == "" {
-		return "", fmt.Errorf("unable to get ID from empty path")
+// RemoveNATRule removes NAT rule and handles task. Return errors when issue risen.
+// Old functions RemoveNATPortMapping and RemoveNATMapping removes using rule details and expects interface is external network.
```

Comment:
"removes using rule details and expects interface is external network" -> "removed using rule details and expected interface to be of external network type."

Reply:
changed